packages/terra-popup/tests/nightwatch/popup-spec.js

```diff
@@ -70,7 +70,7 @@ module.exports = resizeTo(['tiny', 'huge'], {
     browser.url(`${browser.launchUrl}/#/tests/popup-tests/bounded`);
     browser.expect.element('.test-content').to.be.present;
     browser.expect.element('.test-arrow').to.not.be.present;
-    browser.expect.element('svg').to.be.present;
+    browser.expect.element('span').to.be.present;
   },
 
   'Displays a bounded popup without a header': (browser) => {
```

Comment:
Looking at the screenshots below, it looks as if the popups are getting cut off

Reply:
Are you refereeing to `Bounded_Popup[--terra-popup-content-close-icon].838x768.png`and `Bounded_Popup[--terra-popup-content-header-border-bottom].838x768.png`. Those are bound with in a box to show how the popup displays in a width constrained area.